pkgs/tools/security/bitwarden-cli/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, nodejs }:
+
+stdenv.mkDerivation rec {
+  name = "bitwarden-cli-${version}";
+  version = "v1.6.0";
+
+  src = fetchFromGitHub {
+    owner = "bitwarden";
+    repo = "cli";
+    rev = "${version}";
+    fetchSubmodules = true; 
+    sha256 = "1csdbv692l69lickzzl8246gvl0gi58hc91aja3xmb1kcxiphc7c";
+  };
+
+  buildInputs = [ nodejs ];
+  buildCommand = ''
+    unpackPhase  
+    cd "$sourceRoot"
+    export HOME="."
+
+    npm install
```

Comment:
This doesn't work with sandbox enabled:

```
❯ nix-build . -A bitwarden-cli
these derivations will be built:
  /nix/store/fl04p4fz7x2nv4xb4cxqlqzrmikgx6ky-bitwarden-cli-v1.6.0.drv
building '/nix/store/fl04p4fz7x2nv4xb4cxqlqzrmikgx6ky-bitwarden-cli-v1.6.0.drv'...
unpacking source archive /nix/store/db36n4ifdw3qlygqaj9z8b3v1ifnhs0l-source
source root is source
npm ERR! cb() never called!

npm ERR! This is an error with npm itself. Please report this error at:
npm ERR!     <https://npm.community>

npm ERR! A complete log of this run can be found in:
npm ERR!     /build/source/.npm/_logs/2018-12-07T10_35_36_167Z-debug.log
builder for '/nix/store/fl04p4fz7x2nv4xb4cxqlqzrmikgx6ky-bitwarden-cli-v1.6.0.drv' failed with exit code 1
error: build of '/nix/store/fl04p4fz7x2nv4xb4cxqlqzrmikgx6ky-bitwarden-cli-v1.6.0.drv' failed
```

Reply:
npm is used to bring in node dependencies (npm install).  The wiki states that sandbox mode removes internet access, so this will not work.

Will this prevent this package being able to merge?

Reply:
Yes. You’ll need to use Nixpkgs’s node infrastructure to pull in the dependencies purely. Have a look at other node packages to get an idea of how to do this. The manual should help too.